src/MuTalk-Model/MTCompositeTestFilter.class.st

```diff
@@ -0,0 +1,57 @@
+Class {
+	#name : 'MTCompositeTestFilter',
```

Comment:
Could you add a class comment? Does this filter filters if ALL filters say so, or if ANY filter does it? I assume it should be ANY (otherwise it will be unusable, right?)

Reply:
It filters if any filter does. It's as if the tests collection passes through each filter one by one.
I'll add a class comment